packages/plugins/src/sample-project/sample-contracts.ts

```diff
@@ -44,17 +44,11 @@ contract HelloSphinxScript is Sphinx, Script {
     HelloSphinx helloSphinx;
 
     function configureSphinx() public override {
-        sphinxConfig.owners = [${owner}];
-        sphinxConfig.orgId = "${orgId}";
-        sphinxConfig.threshold = 1;
         sphinxConfig.projectName = "My_First_Project";
     }
 
     function run() public sphinx {
-        // Set the \`CREATE2\` salt to be the hash of the owner(s). Prevents
-        // address collisions.
-        bytes32 salt = keccak256(abi.encode(sphinxConfig.owners));
-        helloSphinx = new HelloSphinx{ salt: salt }("Hi", 2);
+        helloSphinx = new HelloSphinx("Hi", 2);
```

Comment:
This is a rider, I updated the sample project to just use create instead of create2.